tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -938,9 +935,18 @@ class MklConvOp : public OpKernel {
           dilations, strides;
       memory::dims dst_dims_tf_order, dst_dims_mkl_order;
 
-      // If pad with conv2d fusion is enabled
-      if (fuse_pad_) {
-        PadWithConvFusion(context, padding_left, padding_right);
+      // For Quantized-Conv2D and Pad fusion, we get padding from the
+      // `padding_list` attribute. Otherwise, we get it from one of the inputs.
+      bool quantized_pad_enabled = false;
+      for (auto const& padding_val : padding_list_) {
+        if (!padding_val) continue;
+        quantized_pad_enabled = true;
+        break;
```

Comment:
Agree. Changed it per your suggestion.

Reply:
@penpornk Just for my learning purposes, can you help me understand how having a `continue` results in more jumps? Any pointers to material would also help. Thanks!

Reply:
Sorry about the confusion! The codes should have the same number of jump instructions. I was just thinking if one might need to jump less based on what values padding_val usually have. I should have added "probably" because I didn't try to count properly and was just speculating what padding_list_ would be like.
```
  compare_condition
  jump_to_"false_label"_if_comparison_is_false
  do_something_if_true
false_label:
  commands_outside_if
```

Reply:
No worries. Thanks for the explanation!